h/accounts/__init__.py

```diff
@@ -96,3 +100,8 @@ def includeme(config):
     config.include('.schemas')
     config.include('.subscribers')
     config.include('.views')
+
+    secret = config.registry.settings['secret_key']
+    derived = derive_key(secret, b'h.accounts')
```

Comment:
But we have a codebase that we're trying to prepare for Python 3 compatibility. These are bytestrings and must remain bytestrings when interpreted by Python 3, so this is correct.